src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ErrorResponse.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.professionalapi.controller.advice;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+public class ErrorResponse {
+
+    private String errorMessage;
+
```

Comment:
ErrorResponse class should have 5 variables in the master code so please verify once and get the changes from master.

Reply:
Removed this class